editors/sc-ide/core/sc_server.cpp

```diff
@@ -378,8 +393,12 @@ void ScServer::onScLangReponse( const QString & selector, const QString & data )
     static QString unmutedSelector("serverUnmuted");
     static QString ampSelector("serverAmp");
     static QString ampRangeSelector("serverAmpRange");
-	static QString startDumpOSCSelector("dumpOSCStarted");
-	static QString stopDumpOSCSelector("dumpOSCStopped");
+    static QString startDumpOSCSelector("dumpOSCStarted");
+    static QString stopDumpOSCSelector("dumpOSCStopped");
+    static QString startRecordingSelector("recordingStarted");
+    static QString pauseRecordingSelector("recordingPaused");
+    static QString stopRecordingSelector("recordingStopped");
+    static QString recordingDurationSelector("recordingDuration");
```

Comment:
I had seen that. I don't see why I get the error though.

````const QStringLiteral synthServerCategory(tr("Sound Synthesis Server"));````

does nothing of the form ````QString xxx = QStringLiteral("yyy")````.

At least I don't recognize that form in it.